service/internal/bootstrap/resources.go

```diff
@@ -43,7 +43,14 @@ func (b *Bootstrap) createAutomationServiceAccount(ctx context.Context) error {
 	} else if err != nil {
 		return microerror.Mask(err)
 	} else {
-		b.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("service account %#q already exists", automationSA.Name))
+		b.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("updating %#q service account", automationSA.Name))
+
+		_, err := b.k8sClient.CoreV1().ServiceAccounts(key.DefaultNamespaceName).Update(ctx, automationSA, metav1.UpdateOptions{})
```

Comment:
I'm wondering if that won't cause conflicts as you probably don't have resourceVersion set here. But I may be wrong.

Reply:
Tested, it just updates sa without issues